src/main/java/com/mitchej123/hodgepodge/HodgepodgeLateMixins.java

```diff
@@ -0,0 +1,34 @@
+package com.mitchej123.hodgepodge;
```

Comment:
why not put this in the mixin package ?

Reply:
The mixin package "owns" any code there, and I believe it intercepts any attempts to instantiate objects from there.

Reply:
```
    /**
     * Package containing all mixins. This package will be monitored by the
     * transformer so that we can explode if some dummy tries to reference a
     * mixin class directly.
     */
     ```
     https://github.com/SpongePowered/Mixin/blob/72d0812084ff2b223f8c11def18158bfa4ad4421/src/main/java/org/spongepowered/asm/mixin/transformer/MixinConfig.java#L250